deepchem/models/tensorgraph/tensor_graph.py

```diff
@@ -19,610 +21,647 @@
 
 class TensorGraph(Model):
 
-  def __init__(self,
```

Comment:
Didn't change too much. Just added some crude methods so that it printed out my beta and alpha share values at given epochs. The majority of the diff came from weird formatting.